src/asm_files.cpp

```diff
@@ -164,3 +161,18 @@ vector<raw_program> read_elf(const std::string& path, const std::string& desired
     }
     return res;
 }
+
+void print_map_descriptors() {
```

Comment:
We link this as a library into binaries with no UI (e.g. user mode daemon), so it would be nice to keep things that depend on cout in separate files from things that don't so the linker can omit them.   Another option would be to make this take the stream as input and have the caller pass cout to it.